standards/snmp/src/v1.rs

```diff
@@ -82,3 +83,111 @@ pub struct VarBind {
     pub name: ObjectName,
     pub value: ObjectSyntax,
 }
+
+#[cfg(test)]
+mod tests {
+    use super::{Message, Trap, VarBind};
+    use alloc::{string::String, string::ToString, vec, vec::Vec};
+    use rasn::types::ObjectIdentifier;
+    use smi::v1::{Gauge, IpAddress, NetworkAddress, TimeTicks};
+
+    fn string_oid(oid: impl AsRef<[u32]>) -> String {
+        oid.as_ref()
+            .iter()
+            .map(ToString::to_string)
+            .collect::<Vec<_>>()
+            .join(".")
+    }
+
+    #[test]
+    fn trap() {
+        #[cfg_attr(rustfmt, rustfmt_skip)]
+        let decode_data = [
+            0x30, 0x4f, // SEQUENCE -> Message
```

Comment:
Could this be formatted with the comments above the data rather than beside the ? I’m not a fan of putting comments on the same line.

Reply:
Sure, done. I left the indentation for the nesting structure but I can also remove that if you prefer.